modules/queue/workergroup.go

```diff
@@ -143,7 +146,11 @@ func (q *WorkerPoolQueue[T]) doStartNewWorker(wp *workerGroup[T]) {
 		log.Debug("Queue %q starts new worker", q.GetName())
 		defer log.Debug("Queue %q stops idle worker", q.GetName())
 
+		atomic.AddInt32(&q.workerStartedCounter, 1)
```

Comment:
When this variable will not be decreased?

Reply:
It never decreases. It's just for debug purpose.

There are other variables like `workerMaxNum` and `workerActiveNum`, they increases/decreases.

Reply:
We should probably document this
```suggestion
		atomic.AddInt32(&q.workerStartedCounter, 1) // Only increase counter, used for debugging
```